src/gc/gc.cpp

```diff
@@ -2083,9 +2083,9 @@ uint8_t* gc_heap::pad_for_alignment_large (uint8_t* newAlloc, int requiredAlignm
 #define END_SPACE_AFTER_GC (LARGE_OBJECT_SIZE + MAX_STRUCTALIGN)
 
 #ifdef BACKGROUND_GC
-#define SEGMENT_INITIAL_COMMIT (2*OS_PAGE_SIZE)
+#define SEGMENT_INITIAL_COMMIT (2*GCToOSInterface::GetPageSize())
```

Comment:
I tend to agree with @Maoni0 that `OS_PAGE_SIZE` looked better, in the GC code at least.